src/database/mountainsData.js

```diff
@@ -0,0 +1,108 @@
+const mountainsData = [
+  {
+    name: 'Greater Chimgan',
+    latitude: 41.4951900,
+    longitude: 70.0586718,
+    elevation: 3309.6,
+    image: 'mnt/mnt_01.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Karakush',
+    latitude: 41.2477890,
+    longitude: 70.3391467,
+    elevation: 3864.3,
```

Comment:
- [ ] As explain by the previous code reviewers , as for the API you used , all the data that is used in the application is to be got from an external API and not stored locally. I will also suggest taking a look at these [mountains  APIs](https://rapidapi.com/search/mountains) and  and update your project 👍 

Keep rocking 🚀